src/message/connection-endpoint.js

```diff
@@ -54,7 +54,7 @@ module.exports = class ConnectionEndpoint extends events.EventEmitter {
     this._ws.on('error', this._onError.bind(this))
     this._ws.on('connection', this._onConnection.bind(this))
 
-    this._authenticatedSockets = []
+    this._authenticatedSocketsCounter = 0
```

Comment:
Given this gets incremented during the authentication callback makes sense to have it as `authenticatedSocketsCounter`